smithy-codegen-core/src/main/java/software/amazon/smithy/codegen/core/directed/CodegenDirector.java

```diff
@@ -266,7 +266,6 @@ public void run() {
         preprocessModelWithIntegrations(integrations);
 
         ServiceShape serviceShape = model.expectShape(service, ServiceShape.class);
-        Set<Shape> shapes = new Walker(model).walkShapes(serviceShape);
```

Comment:
moving this inside `generateShapesInService` should not affect anything, as none of the logic in between updates the `model`.